pages/android/bugreportz.md

```diff
@@ -0,0 +1,21 @@
+# bugreportz
+
+> Generate a zipped Android bug report.
+> This command can only be used through `adb shell`.
+> More information: N/A.
```

Comment:
```suggestion
> More information: <https://android.googlesource.com/platform/frameworks/native/+/master/cmds/bugreportz/>.
```

Reply:
Well, better than nothing. Let's see if anyone has an opinion on this tough.

Reply:
Agreed. Better than nothing - they should really document this properly.